templates/themes/maison/pagination/default_content_pagination.html.twig

```diff
@@ -0,0 +1,8 @@
+<div class="content-pagination">
```

Comment:
Remark not for this line, but for file location (`templates/themes/maison/pagination/default_content_pagination.html.twig`). If it's default I'd rather expect it in the `standard` theme directory.

Reply:
I definitely agree. Especially if we want to show multi-site configuration with shared templates